src/base/bittorrent/session.cpp

```diff
@@ -1139,9 +1142,10 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
 #endif
 
     if (m_listenInterfaceChanged) {
-        const ushort port = this->port();
-        const std::pair<int, int> ports(port, port);
-        settingsPack.set_int(libt::settings_pack::max_retry_port_bind, ports.second - ports.first);
+        const int port = useRandomPort() ? 0 : this->port();
+        if (port > 0)  // user specifies port
```

Comment:
Typo on "specifies"

Reply:
pardon, I don't get it.

Reply:
I think @Piccirello means "user specified (the) port".

Reply:
Thanks, fixed.